### PR TITLE
Added Hardware revision to support Monocam

### PR DESCRIPTION
**Pull request by psmithcrl**
Added Monocam as Hardware Rev 11.

### CODE REVIEW

File: source/LibMultiSense/MultiSenseTypes.hh [hunk 1]
```diff
@@ -2657,6 +2657,7 @@ public:
     static CRL_CONSTEXPR uint32_t HARDWARE_REV_MULTISENSE_S30      = 8;
     static CRL_CONSTEXPR uint32_t HARDWARE_REV_MULTISENSE_S7AR     = 9;
     static CRL_CONSTEXPR uint32_t HARDWARE_REV_MULTISENSE_KS21     = 10;
+    static CRL_CONSTEXPR uint32_t HARDWARE_REV_MULTISENSE_MONOCAM  = 11;
     static CRL_CONSTEXPR uint32_t HARDWARE_REV_BCAM                = 100;
     static CRL_CONSTEXPR uint32_t HARDWARE_REV_MONO                = 101;
```
Comment (psmithcrl) [inline source/LibMultiSense/MultiSenseTypes.hh:2662]: @mattalvarado, should we just use this for monocam?

**Comment by mattalvarado** [comment]
Looks good to me. Is there a function which translates the wire, types to the main interface types which also needs to be updated? Sorry I am out of the office today

**Comment by psmithcrl** [comment]
> Looks good to me. Is there a function which translates the wire, types to the main interface types which also needs to be updated? Sorry I am out of the office today

@mattalvarado Thanks, yes looks like I was missing that.

**Review by mattalvarado — APPROVED** [review]
Looks good to me!

**Pull request merged**